src/components/TaskModule/utils/index.ts

```diff
@@ -13,24 +16,34 @@ export const openNewPageWaitPageVisible = (
     // web.telegram.org will destroy the page when openTelegramLink
     // so send complete request before open link
     if (window?.Telegram?.WebApp?.platform?.includes("web")) {
-      onFinish?.();
+      return req()
+        .then(() => {
+          window?.Telegram?.WebApp?.openTelegramLink?.(url);
+          return true;
+        })
+        .catch(() => {
+          window?.Telegram?.WebApp?.openTelegramLink?.(url);
+          return false;
+        });
```

Comment:
what are you trying to do?

Reply:
This code comes from tomorrowDao, Req calls the interface to complete the task. If it succeeds, it returns true, if it fails, it returns false.